book_source/06_reference/10_editing_this_book.Rmd

```diff
@@ -0,0 +1,62 @@
+# Editing this book {#bookediting}
+
+The file organization of this documentation can be described simply as follows:
+
+- Each **section** is in its own directory.
+- Each **chapter** is in its own file (within the corresponding section).
+
+Sections and chapters are rendered (and numbered) in alpha-numerical order of their corresponding file names.
+Therefore, each section directory and chapter file name should be **prefixed with a two-digit (zero-padded) number**.
+File and directory names should be as similar as possible to the name of the corresponding chapter or section.
+For instance, the file name for this chapter's source file is `01_introduction/03_how_to_use_book.Rmd`.
```

Comment:
Am I misunderstanding how the files are knit together, or should this be `06_reference/10_editing_this_book.Rmd`?